recipes/django/meta.yaml

```diff
@@ -0,0 +1,233 @@
+{% set version = "1.8.11" %}
+
+package:
+    name: django
+    version: {{ version }}
+
+source:
+    fn: Django-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/D/Django/Django-{{ version }}.tar.gz
+    md5: 1c85d0d582dc211adc6851dd2dc86228
+    patches:
+        - libgdal.patch
```

Comment:
I am not sure if this can be fixed on the conda side (maybe I am not smart enough to find a clear solution for this). If we don't patch it, and users have a system and a conda gdal installed, things start to break. I think the best approach to ship a stable package is to "tie" each wrapper to the c-lib we tested (and shipped) with conda.

(BTW patches like this are common in the Linux world because different distros sometimes move files to unexpected places. I lost count of how many patches like this I wrote for Fedora and OpenSUSE.)